src/filesystem.cpp

```diff
@@ -309,44 +310,47 @@ AsyncIOContext *FileAccess::asyncfopen(string *f)
 
 bool FileAccess::asyncopenf()
 {
+    numAsyncReads++;
     if (!localname.size())
     {
         return true;
```

Comment:
The answer is _yes_ 😄 
The first time that `asyncopenf` is called for a `FileAccess` object, the flag must be _false_ so the file can be opened. Next times the flag should be already _true_ to prevent that the file is opened several times (few lines below).